pandas/tests/series/accessors/test_dt_accessor.py

```diff
@@ -259,9 +259,10 @@ def test_dt_accessor_limited_display_api(self):
         tm.assert_almost_equal(results, sorted(set(ok_for_dt + ok_for_dt_methods)))
 
         # Period
-        ser = Series(
-            period_range("20130101", periods=5, freq="D", name="xxx").astype(object)
-        )
+        with tm.assert_produces_warning(FutureWarning, match="Dtype inference"):
+            ser = Series(
+                period_range("20130101", periods=5, freq="D", name="xxx").astype(object)
```

Comment:
can you construct the index outside the context so it is obvious where the warning comes from

Reply:
moved